metrics.go

```diff
@@ -49,6 +49,10 @@ type aggregatingCollector struct {
 	conns         map[uint64] /* id */ *tracingConn
 	rtts          prometheus.Histogram
 	connDurations prometheus.Histogram
+	segsSentDesc  *prometheus.Desc
+	segsRcvdDesc  *prometheus.Desc
+	bytesSentDesc *prometheus.Desc
+	bytesRcvdDesc *prometheus.Desc
```

Comment:
Unlike metrics these are global constants, right? Unlike global mutable variables (global metrics) global constants don't need to be scoped to the `aggregatingCollector` `struct` but instead could just be defined at the module level I suppose. Sorry if my earlier "no globals" rant was confusing. I might as well be missing something Golang idiomatic here.

Reply:
Good point, the descriptions are actually globals. I'll move them out of the struct.